modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/impl/TestMvTableStorage.java

```diff
@@ -40,50 +37,27 @@
  * Test table storage implementation.
  */
 public class TestMvTableStorage implements MvTableStorage {
-    private final Map<Integer, MvPartitionStorage> partitions = new ConcurrentHashMap<>();
+    private final Map<Integer, Storages> storagesByPartitionId = new ConcurrentHashMap<>();
 
-    private final Map<Integer, MvPartitionStorage> backupPartitions = new ConcurrentHashMap<>();
-
-    private final Map<UUID, SortedIndices> sortedIndicesById = new ConcurrentHashMap<>();
-
-    private final Map<UUID, HashIndices> hashIndicesById = new ConcurrentHashMap<>();
+    private final Map<Integer, Storages> backupStoragesByPartitionId = new ConcurrentHashMap<>();
 
     private final TableConfiguration tableCfg;
 
     private final TablesConfiguration tablesCfg;
 
     /**
-     * Class for storing Sorted Indices for a particular partition.
+     * Storages container for the partition.
      */
-    private static class SortedIndices {
-        private final SortedIndexDescriptor descriptor;
-
-        final Map<Integer, SortedIndexStorage> storageByPartitionId = new ConcurrentHashMap<>();
-
-        SortedIndices(SortedIndexDescriptor descriptor) {
-            this.descriptor = descriptor;
+    private static class Storages {
```

Comment:
Would it be better to store indexes in the "TestMvPartitionStorage" instance? What's the ultimate purpose of a separate class?

Reply:
It's just clearer and we change the existing code less, if you insist, I can move it to `TestMvPartitionStorage`.

Reply:
Discussed to do it through delegates so that there are less problems when we change storages. 
Therefore, move `TestHashIndexStorage`, `TestSortedIndexStorage` to `TestMvPartitionStorage`.

Reply:
Fix it

Reply:
Tried to chage it.